Managers/LuaMan.cpp

```diff
@@ -403,6 +405,38 @@ namespace RTE {
 		return isDefined;
 	}
 
+	/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	const std::vector<std::string> & LuaMan::DirectoryList(const std::string &filePath) {
+		std::string fullPath = System::GetWorkingDirectory() + filePath;
+
+		m_Paths.clear();
+
+		for (const auto& entry : std::filesystem::directory_iterator{fullPath}) {
```

Comment:
Should be `auto &entry`. Also, maybe a better name than entry haha

Reply:
But still keep the const?

Reply:
Yep, this is just a matter of moving the `&` from the left to the right for consistency.